app/renderer/components/sectionTitle.js

```diff
@@ -0,0 +1,131 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+const React = require('react')
+const ImmutableComponent = require('../../../js/components/immutableComponent')
+
+const {StyleSheet, css} = require('aphrodite/no-important')
+const globalStyles = require('./styles/global')
+const commonStyles = require('./styles/commonStyles')
+
+class SectionTitleWrapper extends ImmutableComponent {
```

Comment:
what if we create new folder in `renderer` folder named `styles` and we place all styling components into it (one component per file). In there we would keep the same folder structure as we have in `components` folder. This way we keep the context of it. What do you think?

This is one option. Another one is to make exception for styling components and keep them in one file (I still prefer splitting them)

cc @bbondy @bsclifton @cezaraugusto